rosbot_description/urdf/wheel.urdf.xacro

```diff
@@ -1,35 +1,62 @@
+
+      <xacro:property name="ixx" value="0.00007" />
+      <xacro:property name="iyy" value="0.0001263" />
+      <xacro:property name="izz" value="0.00007" />
+
+      <xacro:property name="wheel_separation_y" value="0.192" />
+    </xacro:unless>
 
     <xacro:if value="${prefix == 'fl'}">
       <xacro:property name="x" value="${wheel_separation_x/2}" />
-      <xacro:property name="y" value="${wheel_separation_y/2 - wheel_thickness/2}" />
-      <xacro:property name="y_correction" value="${wheel_thickness/2 - 0.005}" />
-      <xacro:property name="rot" value="${pi/2}" />
+      <xacro:property name="y" value="${wheel_separation_y/2}" />
+      <xacro:property name="mesh" value="file://$(find rosbot_description)/meshes/${wheel_file}_b.dae" />
+      <xacro:property name="visual_rotation" value="${pi}" />
+      <xacro:property name="fdir" value="1 -1 0" />
     </xacro:if>
     <xacro:if value="${prefix == 'fr'}">
       <xacro:property name="x" value="${wheel_separation_x/2}" />
-      <xacro:property name="y" value="-${wheel_separation_y/2 - wheel_thickness/2}" />
-      <xacro:property name="y_correction" value="-${wheel_thickness/2 - 0.005}" />
-      <xacro:property name="rot" value="${-pi/2}" />
+      <xacro:property name="y" value="${-wheel_separation_y/2}" />
+      <xacro:property name="mesh" value="file://$(find rosbot_description)/meshes/${wheel_file}_a.dae" />
+      <xacro:property name="visual_rotation" value="${pi}" />
+      <xacro:property name="fdir" value="1 1 0" />
     </xacro:if>
     <xacro:if value="${prefix == 'rl'}">
-      <xacro:property name="x" value="-${wheel_separation_x/2}" />
-      <xacro:property name="y" value="${wheel_separation_y/2 - wheel_thickness/2}" />
-      <xacro:property name="y_correction" value="${wheel_thickness/2 - 0.005}" />
-      <xacro:property name="rot" value="${pi/2}" />
+      <xacro:property name="x" value="${-wheel_separation_x/2}" />
+      <xacro:property name="y" value="${wheel_separation_y/2}" />
+      <xacro:property name="mesh" value="file://$(find rosbot_description)/meshes/${wheel_file}_a.dae" />
+      <xacro:property name="visual_rotation" value="${0.0}" />
+      <xacro:property name="fdir" value="1 1 0" />
     </xacro:if>
     <xacro:if value="${prefix == 'rr'}">
-      <xacro:property name="x" value="-${wheel_separation_x/2}" />
-      <xacro:property name="y" value="-${wheel_separation_y/2 - wheel_thickness/2}" />
-      <xacro:property name="y_correction" value="-${wheel_thickness/2 - 0.005}" />
-      <xacro:property name="rot" value="${-pi/2}" />
+      <xacro:property name="x" value="${-wheel_separation_x/2}" />
+      <xacro:property name="y" value="${-wheel_separation_y/2}" />
+      <xacro:property name="mesh" value="file://$(find rosbot_description)/meshes/${wheel_file}_b.dae" />
+      <xacro:property name="visual_rotation" value="${0.0}" />
+      <xacro:property name="fdir" value="1 -1 0" />
     </xacro:if>
```

Comment:
There is no decimal point after each value. It is not according to our URDF guide